portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -29,7 +29,9 @@
 import java.util.ArrayList;
 import java.util.List; 
 
-/** Servlet that deletes the entity that has the corresponding ID. */
```

Comment:
(No change required) You don't have to change the comment format here, one line also works. Just when it's a multi-line comment, the sentence should start at the second line.

Reply:
oh okay, I changed it to be consistent with the others. Thanks for letting me know!